### PR TITLE
add Base network

### PR DESCRIPTION
**Pull request by q1q0**
Add connectors addresses on Base network

### CODE REVIEW

File: src/addresses/base/connectorsV2_M1.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+export const connectorsV2_M1 = {
+  '1INCH-A': '0x1D2663d4E2a58323ae63CC571375934AD9c993EC',
+  '1INCH-V5-A': '0x1D2663d4E2a58323ae63CC571375934AD9c993EC',
+  '1INCH-V4-A': '0x16ac1e894abb854519243e9ff982673ab5497549',
+  '1INCH-V3-A': '0x235fca310ac7be45c7ad45f111203468743e4b7c',
```
Comment (shriyatyagii) [inline src/addresses/base/connectorsV2_M1.ts:5]: We don't have one inch on base yet

File: src/addresses/base/connectorsV2_M1.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  'BASIC-C': '0x8De45b985A62639eCfC4c8A898E65426b6cBeE07',
+  'ZEROX-A': '0x426D8cC10ebbC9c6cB4d013209CdcAb46a51eD1a',
+  'ZEROX-V4-A': '0x426D8cC10ebbC9c6cB4d013209CdcAb46a51eD1a',
+  'UNISWAP-V3-ROUTER-A': '0x1852C8b7FD63D8D8749cA8DBd98d9fad9Ccc7AC7',
+  'UNISWAP-V3-SWAP-A': '0xDF1B85eDBbD6DC34713dFF3c0400AD08292d1B70',
+  'DSA-SPELL-A': '0x3B46E32D158db274769601fF9fA306a2c7dfcdd0',
+  'SWAP-AGGREGATOR-A': '0xB914F104dbBd43bc71BE3CaaD97F9bd6f256E67D',
+  'INSTA-DEX-SIMULATION-A': '0x8b69e301EC529A96898a4b386E094D60F241a608',
+  'COMPOUND-V3-A': '0x832925F26de92062fca39262aC90939Ed508325b',
+  'COMPOUND-V3-REWARDS-A': '0x44aC5eD26d8B7BBcF3Bcb9Ac486191cBb294B140',
```
Comment (shriyatyagii) [inline src/addresses/base/connectorsV2_M1.ts:20]:
```suggestion
  'COMPOUND-V3-REWARDS-A': '0x2360Ac6C1F061c5A5Af797FD195bA6B00b3d48Ee',
```

Comment (shriyatyagii) [reply src/addresses/base/connectorsV2_M1.ts:20]: Wrong address

File: src/addresses/base/connectorsV2_M1.ts (hunk 1 above)
Comment (q1q0) [reply src/addresses/base/connectorsV2_M1.ts:5]: Oh, it's my bad....
I just copied them from mainnet and forgot to remove them.

**Review by shriyatyagii — APPROVED** [review]

**Pull request merged**